models/_sector/alm/alm_base_trades.sql

```diff
@@ -0,0 +1,67 @@
+{{ config(
+    schema = 'alm'
+    , alias = 'base_lps'
```

Comment:
agreed. since we are joining to metadata such as prices & erc20 tokens in the macros, i think we can leave out the `base_` naming standard to avoid confusion. we should keep that reserved for dbt lineages where only raw / decoded are included, then later enriched with metadata.

the alternative would be to actually rework the macros to remove metadata & have the `base_` setup, then the final spell can enrich the raw data with metadata, similar to dex trades lineage.